solver/src/liquidity.rs

```diff
@@ -198,6 +209,50 @@ impl Settleable for WeightedProductOrder {
     }
 }
 
+impl Settleable for StablePoolOrder {
+    type Execution = AmmOrderExecution;
+
+    fn settlement_handling(&self) -> &dyn SettlementHandling<Self> {
+        &*self.settlement_handling
+    }
+}
+
+#[derive(Clone, Debug)]
+pub enum BalancerOrder {
+    Weighted(WeightedProductOrder),
+    Stable(StablePoolOrder),
+}
```

Comment:
Should we make these separate values in the Liquidity enum directly? I feel most code has to treat them differently so we'll end up matching first on liquidity and then again on BalancerOrder.

Reply:
Yea, we could do that. I just found this cleaner for other purposes here. I believe this change will be part of the following refactor discussed here in https://github.com/gnosis/gp-v2-services/pull/998#issuecomment-912253237

Reply:
> Should we make these separate values in the Liquidity enum directly?

I think so as well.